lightmdb/views/contactus.py

```diff
@@ -2,7 +2,7 @@
 from lightmdb.forms import ContactForm
 from lightmdb.models import ContactMessage
 
-contactus = Blueprint('contactus', __name__)
+contactus = Blueprint('contact', __name__)
```

Comment:
o sadece sayfa ismi olduğundan değişti, anlamlı sayfa isimleri olussun diye. Ama view isimlerini de contact yapmak için yukarıda yazdığım dosyalarda da güncellenmesi gerekiyor. biraz uzun işlem. Ve ya sadece url-u application.py-dan contactus yapa bilirsin eğer oyle olmasını istiyorsan.